README.md

```diff
@@ -5,7 +5,7 @@
 </p>
 <p align="center">
     <a href="https://github.com/fredrikhgrelland/vagrant-hashistack/actions?query=workflow%3ACI%2FCD+branch%3Amaster">
-      <img alt="Build" src="https://github.com/fredrikhgrelland/vagrant-hashistack/workflows/CI/CD/badge.svg" />
+      <img alt="Build" src="https://img.shields.io/github/workflow/status/fredrikhgrelland/vagrant-hashistack/CI-CD/master" />
```

Comment:
This badge do not work atm because the new changes in `on_pr_master.yml` is not merged